test/helpers/thesis_helper_test.rb

```diff
@@ -76,4 +76,150 @@ class ThesisHelperTest < ActionView::TestCase
     hybrid_thesis = Thesis.new(degrees: [degrees(:one), degrees(:two)])
     assert_equal true, satisfies_advanced_degree?(hybrid_thesis)
   end
+
+  test 'filter_theses_by_department returns a filtered set of theses' do
+    theses = Thesis.includes(:departments)
+    params[:department] = 'Department of Aeronautics and Astronautics'
+    assert_not_equal theses.count, filter_theses_by_department(theses).count
```

Comment:
This assertion isn't wrong, but I wonder if asserting the filtered set is smaller than the full set would provide additional value?